Userland/Libraries/LibCore/ElapsedTimer.cpp

```diff
@@ -20,13 +20,14 @@ ElapsedTimer ElapsedTimer::start_new()
 void ElapsedTimer::start()
 {
     m_valid = true;
-    m_origin_time = m_precise ? Duration::now_monotonic() : Duration::now_monotonic_coarse();
+    m_origin_time = m_precise ? MonotonicTime::now() : MonotonicTime::now_coarse();
 }
 
 void ElapsedTimer::reset()
 {
     m_valid = false;
-    m_origin_time = {};
+    // Since the timer is not valid, the exact time we set is not relevant.
+    m_origin_time = { MonotonicTime::now_coarse() };
```

Comment:
Couldn't we just leave the old value then? That saves us one (probably inexpensive) `CLOCK_MONOTONIC_COARSE`.

Reply:
That's a good point; I just chose some default since the old code defaulted the time, which would set it to an unspecified value (since the origin point of the monotonic clock is unspecified).